src/configurationProvider.ts

```diff
@@ -1,7 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
+import * as _ from "lodash";
+import * as os from "os";
 import * as vscode from "vscode";
+
```

Comment:
Nice to have extra line to separate 3rd party dependencies and local dependencies.